File: Jan_Basic_Programming/Assignments/C/50.cpp
#include<iostream>
#include<bits/stdc++.h>
using namespace std;

int main()
{
     int rows, n=1;

    cout << "Enter the Number Of rows for Diamond Triangle : ";
    cin >> rows;

    for(int row=1; row<(2*rows); row++){
        if(row<=rows){
            for(int space=1; space<=rows-row; space++){
                cout<<" ";
            }
            for(int star=1; star<=row; star++){
                cout<<"* ";
            }
        }else{
            for(int space=1; space<=row-rows; space++){
                cout<<" ";
            }
            for(int star=1; star<=(2*rows - row); star++){
                cout<<"* ";
            }
        }
        cout<<endl;
    }
    
    return 0;
}

File: Mar_PHP/Assignments/BasicPrograms/50.php
<!DOCTYPE html>
<html>
<head>
    <title>Diamond Triangle</title>
</head>
<body>

<form method="post">
    <label for="rows">Enter the Number Of rows for Diamond Triangle:</label>
    <input type="number" id="rows" name="rows">
    <button type="submit">Submit</button>
</form>

<?php
if ($_SERVER["REQUEST_METHOD"] == "POST") {
    $rows = $_POST['rows'];

    for ($row = 1; $row < (2 * $rows); $row++) {
        if ($row <= $rows) {
            for ($space = 1; $space <= $rows - $row; $space++) {
                echo "&nbsp;&nbsp;";
            }
            for ($star = 1; $star <= $row; $star++) {
                echo "*&nbsp;&nbsp;";
            }
        } else {
            for ($space = 1; $space <= $row - $rows; $space++) {
                echo "&nbsp;&nbsp;";
            }
            for ($star = 1; $star <= (2 * $rows - $row); $star++) {
                echo "*&nbsp;&nbsp;";
            }
        }
        echo "<br>";
    }
}
?>

</body>
</html>
